R/pkg/R/generics.R

```diff
@@ -385,6 +385,7 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 #' @return A SparkDataFrame.
 #' @rdname summarize
 #' @export
+#' @noRd
```

Comment:
hmm, I think some of the problems are orthogonal - for example, to avoid a `avg.rd` file we could just change `avg` in generic.R to have `@rdname column_aggregate_functions`?

I feel the standard is to have documentation on the generic (and have it on the same rd) like https://stat.ethz.ch/R-manual/R-devel/library/base/html/mean.html